engine/source/materials/mat/mat069/sigeps69.F

```diff
@@ -239,6 +215,13 @@ SUBROUTINE SIGEPS69(
           ET(I) = MAX(ONE,ET(I)/GMAX)
         ENDDO
       ENDIF
+c--- avoid buckling
+      P_FAC(1:NEL) = ONE
+      NU_1 = FOURTY*(HALF-(3*RBULK-GMAX)/(6*RBULK+GMAX))
+      DO I=1,NEL
+        AMIN = MIN(EV(I,1),EV(I,2),EV(I,3))
+        IF (AMIN<ZEP2) P_FAC(I) = MAX(ONE,NU_1/AMIN)
```

Comment:
There is a chance of division by 0 here, could you make it more robust?

Reply:
EV(I,1) are the stretch, always >0 (see line 179-181); zero corresponds to 100% compression (zero volume). but you are right, numerically it's possible, i'll add a modif.